packages/shared/src/lib/flows/triggers/trigger.ts

```diff
@@ -14,7 +15,7 @@ const commonProps = {
   name: Type.String({}),
   valid: Type.Boolean({}),
   displayName: Type.String({}),
-  nextAction: Type.Optional(Type.Any()),
+  nextAction: Type.Optional(Action),
```

Comment:
This now fails at runtime
 "Failed building the validation schema for POST: /v1/flow-versions/:flowVersionId/trigger, due to error reference \"T0\" resolves to more than one schema"